lib/workers/global/limits.ts

```diff
@@ -0,0 +1,40 @@
+import { logger } from '../../logger';
+
+const limitsToInit = ['prCommitsPerRunLimit'];
+const l = {};
```

Comment:
Please add typeinfo, eg 
```suggestion
const l: Record<string,numnber> = {};
```